setup.py

```diff
@@ -2,18 +2,22 @@
 import re
 from setuptools import find_packages
 from setuptools import setup
+import io
 
 here = os.path.abspath(os.path.dirname(__file__))
 try:
     # obtain version string from __init__.py
-    init_py = open(os.path.join(here, 'lasagne', '__init__.py')).read()
-    version = re.search('__version__ = "(.*)"', init_py).groups()[0]
+    with io.open(os.path.join(here, 'lasagne', '__init__.py'), 'r') as f:
+        init_py = f.read()
+        version = re.search('__version__ = "(.*)"', init_py).groups()[0]
```

Comment:
Sorry for the nitpick, but the last line should be left unindented as before. It's not part of the file operation. Thanks!